Source/ZoomNet/Resources/Groups.cs

```diff
@@ -1,6 +1,44 @@
+using Pathoschild.Http.Client;
+using System.Text.Json.Nodes;
+using System.Threading;
+using System.Threading.Tasks;
+
 namespace ZoomNet.Resources;
 
-public class Groups
+/// <summary>
+/// Allows you to manage Groups.
+/// </summary>
+public class Groups : IGroups
 {
-	
+	private readonly IClient _client;
+
+	internal Groups(IClient client)
+	{
+		_client = client;
+	}
+
+	/// <summary>
```

Comment:
No need to copy/paste the XML documentation from the interface. You can simply use:
```
	/// <inheritdoc/>
```

Reply:
Don't forget to remove the unnecessary XML comments.